File: fix.py
import re

file = "index.js"

with open(file, "r", encoding="utf-8") as f:
    code = f.read()

# 🔧 arreglar join roto o incompleto
code = re.sub(r"lineas\.join\([^\)]*$", "lineas.join('\\n');", code)

# 🔧 arreglar casos con doble paréntesis
code = code.replace("lineas.join('\\n'))", "lineas.join('\\n')")

# 🔧 arreglar saltos de línea rotos en strings
code = code.replace("lineas.join('\n", "lineas.join('\\n')")

with open(file, "w", encoding="utf-8") as f:
    f.write(code)

print("✔ index.js reparado")

File: fix.js
const fs = require('fs');

const file = 'index.js';

let code = fs.readFileSync(file, 'utf8');

// corregir error de join roto
code = code.replace(/lineas\.join\('\s*\n?/g, "lineas.join('\\n')");

fs.writeFileSync(file, code);

console.log('✔ index.js corregido');
